index.html

```diff
@@ -536,34 +536,54 @@
-automating the recognition of a [=person=]'s identity across different [=contexts=] is
-[=inappropriate=]. This is particularly true for [=vulnerable=] people as recognising them
-in different [=contexts=] may force their vulnerability into the open.
-
-[=Identity=] being by nature fragmented, [=user agents=] must work in support of [=users=]
-having different identities in different [=context=] with respect to <em>all</em> [=parties=]
-(including the user agent vendor) and to prevent their recognition through other means,
-where possible.
-
-A keystone principle of the Web is trust [[RFC8890]]. An important part of trust is to
-ensure that [=data=] collected for a [=purpose=] that matches a clearly delineated [=user=]
-feature should not then be used for additional secondary [=purposes=]. Email is often used
-for login and communication [=purposes=], including essential transactional interactions.
-Using emails for cross-context recognition [=purposes=] is therefore not only
-[=inappropriate=] but also threatens key messaging infrastructure. Future iterations of
-the Web platform should ensure that users can log into sites and receive communication
-without relying on email at all.
+A [=person=]'s <dfn>identity</dfn> is the set of characteristics that define them. Their identity
+*in a [=context=]* is the set of characteristics they choose to present to that context. People
+frequently present different identities to different contexts, and also frequently share an identity
+between several contexts.
+
+<dfn>Cross-context recognition</dfn> is the act of recognising that an [=identity=] in one
+[=context=] is the same [=person=] as an [=identity=] in another [=context=]. [=Cross-context
+recognition=] can at times be [=appropriate=] but anyone who does it needs to be careful not to
+apply the [=norms=] of one [=context=] in ways that violate the [=norms=] around use of information
+acquired in a different [=context=]. (For example, if you meet your therapist at a cocktail party,
+you expect them to have rather different discussion topics with you than they usually would, and
+possibly even to pretend they do not know you.) This is particularly true for [=vulnerable=] people
+as recognising them in different [=contexts=] may force their vulnerability into the open.
+
+In computer systems and on the Web, an [=identity=] seen by a particular website is typically
+assigned an <dfn>identifier</dfn> of some type, which makes it easier for an automated system to
+store data about that user.
+
+[=User agents=] support their [=users=]' [=autonomy=] by helping them present their intended
+[=identity=] to each [=context=] that they visit. To do this, [=user agents=] have to make some
+assumptions about the borders between [=contexts=]. By default, [=user agents=] define a
+<dfn>machine-enforceable context</dfn> or <dfn>partition</dfn> as:
+
+* A set of [=environments=] (roughly top-level pages, iframes, and workers)
```

Comment:
Reorder to emphasize that the iframes and workers don't have to be top-level:

```suggestion
* A set of [=environments=] (roughly iframes (including third-party iframes), workers, and top-level pages,)
```

Reply:
Tweaked.